packages/recoil/core/__tests__/Recoil_perf-test.js

```diff
@@ -26,11 +27,15 @@ const ITERATIONS = [1]; // Avoid iterating for automated testing
 // const ITERATIONS = [10, 100, 1000, 10000];
 // const ITERATIONS = [10, 100, 1000, 10000, 100000];
 
-function testPerf(name: string, fn: number => void) {
+function testPerf(
```

Comment:
how does this test case assure performance is good?

Reply:
It doesn't for automated testing.  But, it's there for at least being able to manually run performance tests.